### ISSUE TITLE
File transfer notes are not cluster agnostic

### ISSUE THREAD

**Issue by f-allian** [rcgsheffield/sheffield_hpc#2037]
- The current docs on how to transfer files to/from the clusters are currently not cluster agnostic, see [here](https://docs.hpc.shef.ac.uk/en/latest/hpc/transferring-files.html#gsc.tab=0)
- Should replace 
    - _Where below substitute $CLUSTER_NAME with Stanage or Bessemer and $USER with your cluster username._
    with something like:
    -  _Where below substitute $CLUSTER_NAME with Stanage ($USERS) and Bessemer ($HOME) with your cluster username._ 
    
 Also from @willfurnass 
 
 - "A suggestion: if we specify relative paths in the scp/rsync examples then files will be transferred into people's home dirs.  That way we can 'correct' for this issue without adding much more content/complexity to that page e.g. `scp /home/user/mything.dat $USER@$CLUSTER_NAME.shef.ac.uk:images`/"

**Comment by jkwmoore** [comment]
This is indeed not agnostic for the clusters which is not ideal. Needs a fix.

>Where below substitute $CLUSTER_NAME with Stanage ($USERS) and Bessemer ($HOME) with your cluster username.

I would advise very strongly against this as this would be a bit misleading. We should/must stick to using standard bash environment variables as what they are i.e. $USER = the username of the current user. $HOME = the home directory of the current user.

Using $USERS and $HOME is likely to cause confusion between this being used as a place holder or as an environment variable since $HOME is one and $USERS is not.

$USER as used inline for the docs at the moment appears to be being used to indicate the name of the user (as the shell env var would contain) for direct parameter expansion.

Using the $USER env var correctly for home directories on each cluster would be:
- Stanage would be `/users/$USER` 
- Bessemer would be `/home/$USER`

Not ideal as again, not cluster agnostic.

We could use tabs here, one for each cluster but this seems a bit clumsy to me as we could replace and use $HOME (as this would always contain the correct home dir path irrespective of the cluster or any linux machine).  

Using $HOME properly would however need hinting that we are truly talking about using the bash env var, that this is not a placeholder to be replaced with your username or otherwise.

Care would also need to be taken given the parameter expansion will occur on the local machine (i.e. the variables populate on what they're set locally). Although this wouldn't be an issue when connecting **between** Sheffield HPCs due to shared usernames, the home directories differ and on other remote systems the usernames may vary.

So, in summary I see two issues here:

- Conflation of bash environment variables and placeholder variables.
- Guidance is not cluster agnostic.

I would either go for using environment variables properly or getting rid of them entirely in favour of placeholder variables not using $ prefix to avoid confusion. This choice informs the solution of making it agnostic, though we may not want to make this agnostic due to the increase in complexity if trying to guide with the use of $HOME
